### PR TITLE
Added home/landing page

### PR DESCRIPTION
**Pull request by Shah-Arsalan**
Added Landing page for the web app. 
Screenshot (259)

### CODE REVIEW

File: TravelBug/src/Components/Sidebar/Sidebar.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.scroller {
+  background-color: var(--primary-color);
+
+  padding: 1rem;
+  height: 100vh;
+  border-right: 1px solid #fff;
```
Comment (ramalingaa) [inline TravelBug/src/Components/Sidebar/Sidebar.css]: Try using variables for color codes

**Review by ramalingaa — APPROVED** [review]
UI looks good. Good folder structure. Clean code. Good to merge

File: TravelBug/src/Components/VideoList/VideoList.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { useData } from "../../Contexts/Datacontext";
+import { VideoCard } from "../VideoCard/VideoCard";
+import "./VideoList.css";
+
+const VideoList = () => {
+  const { state } = useData();
+  const { videos } = state;
+  console.log("in videocard", videos);
```
Comment (vinc3nati) [inline TravelBug/src/Components/VideoList/VideoList.jsx]: remove console logs

**Review by vinc3nati — APPROVED** [review]
UI looks great

**Review by hrishi82 — APPROVED** [review]
Code looks good to me, can be merged!

File: TravelBug/src/Components/Sidebar/Sidebar.css (hunk 1 above)
Comment (Shah-Arsalan) [reply TravelBug/src/Components/Sidebar/Sidebar.css]: Done

File: TravelBug/src/Components/VideoList/VideoList.jsx (hunk 2 above)
Comment (Shah-Arsalan) [reply TravelBug/src/Components/VideoList/VideoList.jsx]: Done

**Pull request merged**